backend/src/main/java/com/google/models/Timestamp.java

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.models;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Wrapper for LocalDate object with day, month, year fields. 
+ * Accepts format in yyyy-MM-dd or yyyy-M-d or yyyy-MM-d or yyyy-M-dd, but prints format in yyyy-MM-dd.
+ */
+public final class Timestamp {
+  public static final String TIMESTAMP_PROPERTY = "timestamp";
+  private static final String EXCEPTION_MESSAGE = "Invalid string format.";
+  private static final DateTimeFormatter dateFormatter = new DateTimeFormatterBuilder()
```

Comment:
if you don't end up modifying it, might be good to consider is as a 'constant' DATE_FORMATTER since it is expected to remain the same for all code in this class.

Reply:
i see, i guess i didn't really think of objects as constants.